commands/team.go

```diff
@@ -335,21 +336,23 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	if public {
 		privacy = model.TeamOpen
 	}
-
+	var Error error
 	teams := getTeamsFromTeamArgs(c, args)
 	for i, team := range teams {
 		if team == nil {
+			Error = multierror.Append(Error, fmt.Errorf("Unable to find team '" + args[i] + "'"))
 			printer.PrintError("Unable to find team '" + args[i] + "'")
 			continue
 		}
 		if updatedTeam, _, err := c.UpdateTeamPrivacy(team.Id, privacy); err != nil {
+			Error = multierror.Append(Error, fmt.Errorf("Unable to modify team '" + team.Name + "' error: " + err.Error()))
 			printer.PrintError("Unable to modify team '" + team.Name + "' error: " + err.Error())
 		} else {
 			printer.PrintT("Modified team '{{.Name}}'", updatedTeam)
 		}
 	}
 
-	return nil
+	return Error
```

Comment:
After the change to `*multierror.Error`, you need to call `errors.ErrorOrNil()` to return an `error` here.